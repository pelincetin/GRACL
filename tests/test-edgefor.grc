int main() {
    Graph g = createGraph(100);
    EdgeList el = createEdgeList();
    el.appendEdge(g.addEdge(g.createNode("1"), g.createNode("2"), infinity));
    el.appendEdge(g.addEdge(g.createNode("3"), g.createNode("4"), 3.14));
    for (Edge e in el){
        print(doubleToString(e.weight()));
        print(e.start().data());
        print(e.end().data());
    }
}