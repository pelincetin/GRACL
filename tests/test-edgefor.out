inf
1
2
3.140000
3
4